doc/source/tutorials/instrument.rst

```diff
@@ -110,7 +110,7 @@ complex than the local oscillator ones.
 
 Let's see a minimal example:
 
-.. code-block::  python
+.. testcode::  python
 
```

Comment:
This `testcode` fails because of `proprietary_instruments`. We should probably revert it back to `code-block`.